site/content/faq/what-about-typescript-support.md

```diff
@@ -0,0 +1,5 @@
+---
+question: What about Typescript support?
+---
+
+You need to install a [community supported preprocessor](https://github.com/sveltejs/integrations#preprocessors) such as [svelte-preprocess](https://github.com/kaisermann/svelte-preprocess). Work is ongoing to improve [IDE support](https://github.com/sveltejs/language-tools) and build [additional CLI tooling](https://github.com/sveltejs/language-tools/issues/68)
```

Comment:
A new tracking ticket for IDE support just got created that would be good to use here. I might change the link text "IDE support" to also include the previous work "improve" and then change the link to https://github.com/sveltejs/language-tools/issues/83

I know you're just trying to port the existing FAQ, but I figured it might save us from forgetting to update this link later :-)